src/api/handlers/userprofile.py

```diff
@@ -267,17 +268,29 @@ def complete_imported_user(self, request):
   def handle_contacts_csv(self, request):
     context: Context = request.context
     args: dict = context.args
-    # find the community within the team that the 
-    csv_ref = args['csv'].file 
-    # csv_ref is a bytes object, we need a csv
-    # so we copy it as a csv temporarily to the disk
-    temporarylocation="testout.csv"
-    with open(temporarylocation, 'wb') as out:
-      var = csv_ref.read()
-      out.write(var)
-    info, err = self.service.handle_csv(context, args, temporarylocation)
+    try:
+    
+      # find the community within the team that the 
+      csv_ref = args['csv'].file 
+      # csv_ref is a bytes object, we need a csv
+      # so we copy it as a csv temporarily to the disk
+      temporarylocation="testout.csv"
+      with open(temporarylocation, 'wb') as out:
+        var = csv_ref.read()
+        out.write(var)
+      # filecontents is a list of dictionaries, with each list item representing a row in the CSV
+      filecontents = []
+      with open(temporarylocation, "r") as f:
+          reader = csv.DictReader(f, delimiter=",")
+          for row in reader:
+            filecontents.append(row)
```

Comment:
this logic seems like it would be useful as a helper function.  can you make a helper function in utils/common.py and define this there?